src/Traits/Translatable.php

```diff
@@ -244,6 +245,81 @@ public function setAttributeTranslations($attribute, array $translations, $save
         return $response;
     }
 
+    /**
+     * Gets Entry if has translated field field with value via scope.
+     * 
+     * @example  $class->whereTranslation('de', 'title', 'zuhause') 
+     * 
+     * @param builder       $query
+     * @param string|array  $locale     {required} locale(s) you are looking for the field.
+     * @param string        $field      {required} the field your looking to find a value in.
+     * @param string        $operator   {required} value you are looking for or a relation modifier.
+     * @param string        $value      {optional} value you are looking for. Only use if you supplied an operator.
+     * @param bool          $default    {optional} fi true checks for $value is in default language before checking translations.
+     *
+     * @return Builder|null
+     * 
+     */
+    public function scopeWhereTranslation(Builder $query, $locales, string $field, string $operator, string $value = null, bool $default = false){
```

Comment:
Argument type declarations `string` and `bool` were introduced with PHP 7 but we still support 5.4, ref:
http://php.net/manual/en/functions.arguments.php#functions.arguments.type-declaration
```suggestion
    public function scopeWhereTranslation(Builder $query, $locales, $field, $operator, $value = null, $default = false){
```